frontend/vue/questions/vue.md

```diff
@@ -1,3 +1,12 @@
 # Vue.js
 
-* What's Vue.js?
\ No newline at end of file
+* What's Vue.js?
+* What's Prop component?
+* What's reactivity in Vue.js?
+* What's state management?
+* Describe lifecycle hooks in Vue.js?
+* Describe different between v-if and v-show?
+* What are the most essential elements of the State Management Pattern?
+* Explain how to use $parent?
+* How do you verify if a mutation occurs?
+* What are the benefits of using Vue CLI?
```

Comment:
I think this question needs to be asked in the form of talking about the benefits in certain use cases.

Reply:
In the interview, any question may be asked, in some cases these questions have been seen.